pkgs/development/libraries/libimobiledevice-glue/default.nix

```diff
@@ -1,26 +1,30 @@
-{ lib
-, stdenv
-, fetchFromGitHub
-, autoreconfHook
-, pkg-config
-, libplist
-, nix-update-script
-}:
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+
+  autoreconfHook,
+  pkg-config,
 
-stdenv.mkDerivation rec {
+  libplist,
+  nix-update-script,
```

Comment:
We should add [unstableGitUpdater](https://github.com/NixOS/nixpkgs/pull/105713) or nix-update-script to all the packages so they are more easily kept up to date.
I'm not quite sure what the differences between unstableGitUpdater and nix-update-script are, couldn't find some explanation easily, maybe someone more knowledgeable could help us here, but I saw both used for other unstable packages so it should be fine, I think.

Reply:
Actually a good point. I was thinking about that since some had it. Will try unstableGituUpdater first as it _sounds_ like it may be better suited to libimobiledevice's strange release choice of no tag bumps